index.html

```diff
@@ -12,39 +12,175 @@
     <!--This is the header section-->
     <header>
         <nav>
-            <a href="#"><img src="mylogo.png" alt="Logo" id="myLogo"></a>
+            <a href="#"><img src="req/mylogo.png" alt="Logo" id="myLogo"></a>
             <!--Nav bar button-->
-            <img src="Union.png" alt="Logo" id="logo">
+            <img src="req/Union.png" alt="Logo" id="logo">
         </nav>
     </header>
     <!--This is Headline-->
     <section class="headline">
-        <div id="bigText"><h1>I’m Adil
-            Glad to see you!</h1></div>
-        <div id="smallText">
-            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <div><p id="bigText">I’m Adil<br>
+            Glad to <br>see you!</p></div>
+        <div>
+            <p class="smallText">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
         </div>
         <div class="socialmedia">
-            <div id="supportingText">LET'S CONNECT</div>
+            <div class="supportingText">LET'S CONNECT</div>
             <!--social media icons-->
             <ul>
             <li>
-                <a href="#"><img src="twitter.png" alt="twitter" class="icon icon1"></a>
+                <a href="#"><img src="req/twitter.png" alt="twitter"></a>
             </li>
             <li>
-            <a href="#"><img src="linkedin.png" alt="linkedin" class="icon icon2"></a>
+            <a href="#"><img src="req/linkedin.png" alt="linkedin"></a>
             </li>
             <li>
-            <a href="#"><img src="M.png" alt="M" class="icon icon3"></a>
+            <a href="#"><img src="req/M.png" alt="M"></a>
             </li>
             <li>
-            <a href="#"><img src="github.png" alt="github" class="icon icon4"></a>
+            <a href="#"><img src="req/github.png" alt="github"></a>
             </li>
             <li>
-            <a href="#"><img src="V.png" alt="V" class="icon icon5"></a>
+            <a href="#"><img src="req/V.png" alt="V"></a>
             </li>
```

Comment:
- [ ] [OPTIONAL] If you consider allowing visitors to connect or contact you - which I believe that is why you added the media, please consider inserting your profile link or username on those media platforms in the `href` instead of the `#` that you currently use. This is good for good user experience.